packages/react-components/react-input/library/src/components/Input/useInputStyles.styles.ts

```diff
@@ -45,6 +45,7 @@ const useRootClassName = makeResetStyles({
   borderRadius: tokens.borderRadiusMedium, // used for all but underline
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31812&commitId=ce03a369a3b21a0474b2d0a0568cd9d6fdc9324e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|0| Removed|
|Avatar Converged.basic.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|0| Added|

</details>


  <details><summary><b>Input Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Input Converged.Inline.default.chromium.png|271| Changed|

</details>


  <details><summary><b>SearchBox Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox Converged.Size- small.hover.chromium.png|233| Changed|
|SearchBox Converged.Size- small.focused.chromium.png|233| Changed|
|SearchBox Converged.Size- small.default.chromium.png|217| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>